documentation/index.md

```diff
@@ -572,15 +600,19 @@ assert pipe(0) == (None, None, None)
 ```
 
 
-### Fine Control of Parallel Execution
+### **Fine Control of Parallel Execution**
 
-1. You can control which node is preferred to run 1st when mutliple `ExecNode`s are available for execution. This can be achieved through modifications of `priority` of the `ExecNode`.
-1. You can even make an `ExecNode` run alone (i.e. without allowing other ExecNodes to execute in parallel to it). This can be helpful if you write code that is not thread-safe or use a library that is not thread-safe in a certain `ExecNode`.
-This is achieved by setting the `is_sequential` parameter to `True` for the `ExecNode` in question. The default value is set via the environment variable `TAWAZI_IS_SEQUENTIAL` (c.f. `tawazi.config`).
-1. You can control the behavior of the `DAG` in case an `ExecNode` fails:
-  a. `"strict"`: stop execution of the DAG
-  b. `"all-children"`:  stop the execution of the all successors
-  c. `"permissive"`: continue the execution of the whole DAG
+1. You can control which node is preferred to run 1st when multiple `ExecNode`s are available for execution.
+This can be achieved through modifications of `priority` attribute of the `ExecNode`.
+2. You can even make an `ExecNode` run alone (i.e. without allowing other ExecNodes to execute in parallel to it). This can be helpful if you write code that is not thread-safe or use a library that is not thread-safe in a certain `ExecNode`.
```

Comment:
```suggestion
1. You can even make an `ExecNode` run alone (i.e. without allowing other ExecNodes to execute in parallel to it). This can be helpful if you write code that is not thread-safe or use a library that is not thread-safe in a certain `ExecNode`.
```

Reply:
why do you prefer going back to 1 ?